components/TextFields/src/private/MDCTextInputCommonFundament.m

```diff
@@ -159,6 +161,7 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
         [aDecoder decodeObjectForKey:MDCTextInputFundamentCharacterRelativeSuperviewKey];
     _textColor = [aDecoder decodeObjectForKey:MDCTextInputFundamentTextColorKey];
     _trailingUnderlineLabel = [aDecoder decodeObjectForKey:MDCTextInputFundamentTrailingLabelKey];
+    _trailingViewMode = (UITextFieldViewMode)[aDecoder decodeIntegerForKey:MDCTextInputFundamentTrailingViewKey];
```

Comment:
Do we need to check for the existence of these keys before we decode them?
```
if ([aDecoder containsValueForKey:MDCTrailingViewKey]) {
  _trailingViewMode = (UITextFieldViewMode)[aDecoder decoderIntergerForKey:MDCTrailingViewKey];
}
```
And the other ivars? (Happy to have that in another PR if we do need to do so)